pandas/core/computation/parsing.py

```diff
@@ -91,7 +94,7 @@ def clean_backtick_quoted_toks(tok: Tuple[int, str]) -> Tuple[int, str]:
     return toknum, tokval
 
 
-def clean_column_name(name: str) -> str:
+def clean_column_name(name: "Label") -> "Label":
```

Comment:
when changing annotations can you also ensure the docstring matches.

Reply:
Ah, thanks for catching this.